assets/javascripts/formElement.jsx

```diff
@@ -0,0 +1,58 @@
+var React = require('react');
+var riek = require('riek')
+var RIEInput = riek.RIEInput;
+var FormBank = require('./formBank.js');
+
+module.exports = React.createClass({
+  updateElementText: function(newText, cellId, rowId) {
+    var tableId = this.props.id;
+    if (cellId == undefined) { //i.e. if we are updating the text of a standard form element in builder, not a tableCell, which is form within a Table
+      this.props.updateFormElementText(newText, tableId);
+    } else {
+      this.props.updateTableElementText(newText, cellId, rowId, tableId);
+    }
+  },
+  deleteElement: function(event, id) {
+    event.preventDefault();
+    var id = this.props.id;
```

Comment:
Is there a reason your declaring this `var id` in the following few function? Could you just put `this.props.id` into the arg directly?